lib/jackhammer/topic_manager.rb

```diff
@@ -4,12 +4,13 @@ module Jackhammer
   class TopicManager
     class << self
       def topics
-        topics = {}
+        result = {}
         Jackhammer.configuration.yaml.each do |topic, topic_config|
+          fail(InvalidConfigError, "Topic config is invalid") unless topic_config.is_a?(Hash)
```

Comment:
@scottserok I'm not sure it was here ^, when you reviewed.

Reply:
Ah I very likely missed it. My eyes went straight to the bottom of this method looking for a rescue statement. Thanks for pointing it out!